docs/user_guide/HOWTO/IPA_Clients.rst

```diff
@@ -0,0 +1,86 @@
+.. _howto-ipa_clients:
+
+HOWTO Enroll Hosts into an IPA Domain
+=====================================
+
+Host should be able to join an IPA domain with a few catches. SIMP already uses
+the login stack that IPA uses, but it also actively manages the same resources
+that IPA provides automation for, like:
+
+#. autofs
+#. sudoers
+#. SELinux
+#. krb5
+
+These features may work in the future, but logins via SSSD or LDAP should work
+without issue.
+
+
+Adding clients
+--------------
+
+Adding clients requires two steps:
+
+#. Add the hosts on the IPA server, setting a one time password
+#. Running ``ipa-client-install`` on the host, using the password generated in
```

Comment:
Does this work in FIPS mode on EL6?

Reply:
Do you have the buzilla link to the issue?

Reply:
I was able to join an EL6 vagrant box to my test IPA server

Reply:
Wait that wasn't FIPS mode. Nevermind.

Reply:
Yeah, that was a bit more direct that I was expecting...

```bash
[root@el6 ~]# ipa-client-install --unattended --password=password --server=ipa.test --domain=test --realm=TEST
Cannot install IPA client in FIPS mode
[root@el6 ~]# 
```